model/src/main/kotlin/utils/LicenseResolver.kt

```diff
@@ -41,10 +45,9 @@ internal class LicenseResolver(private val ortResult: OrtResult) {
     fun getDetectedLicensesForId(id: Identifier): SortedSet<String> =
         collectLicenseFindings(id).keys.mapTo(sortedSetOf()) { it.license }
 
-    private fun getPathExcludes(id: Identifier): List<PathExclude> =
-        // Only license findings of projects can be excluded by path excludes.
-        // TODO: Implement path excludes for packages.
-        if (ortResult.isProject(id)) ortResult.getExcludes().paths else emptyList()
+    private fun getPathExcludes(id: Identifier, provenance: Provenance): List<PathExclude> =
+        if (ortResult.isProject(id)) ortResult.getExcludes().paths
+        else packageConfigurationProvider.getPackageConfiguration(id, provenance)?.pathExcludes.orEmpty()
```

Comment:
Use curly braces if the condition does not fit in one line.

Reply:
Done.